git/README.md

```diff
@@ -0,0 +1,44 @@
+# Git
+
+Git is pretty awesome. Here's some reasonable standards when it comes to using it.
+
+## Commit Messages
+
+At Mobify, we recommend a few standards regarding git commit messages:
+- If the commit is related to a JIRA ticket, prefix the case-sensitive ticket number to the start of the message
```

Comment:
We have quite a few different viewpoints on this point:
* Prefix every commit with JIRA number (@m)
* Super-structured commits (@mikenikles)
* Branch names prefixed with JIRA number (@benlast)
* PR title or description includes JIRA number (me)

I think each has a different driver based on the workflow and needs of the person suggesting it. Is it worth laying down a standard that may not fit each team it applies to? 

@jvoll also brought up a good point: what about partners? Do we expect them to follow this standard?